### PR TITLE
Fixes Virology Civilian Bounties.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request

Whooops. Turns out virology bounties aren't covered by reagent bounties, somehow, so here we are.
This does some minor codewise cleanup of civilian bounties so that virologist bounties can actually be completed reasonably. Stat requirements on bounties are the usual requirement, send at least one unit to complete, etc etc.

## Why It's Good For The Game

🐛 🔨 💥 

## Changelog
:cl:
fix: Virology civilian bounties can now be completed again.
/:cl:

### CODE REVIEW

**Comment by Timberpoes** [comment]
<3

File: code/modules/economy/account.dm [hunk 1]
```diff
@@ -133,6 +129,8 @@
 	if(istype(civilian_bounty, /datum/bounty/reagent))
 		var/datum/bounty/reagent/chemical = civilian_bounty
 		return "[chemical.shipped_volume]/[chemical.required_volume] u"
+	if(istype(civilian_bounty, /datum/bounty/virus))
+		return "A least 1 u"
```
Comment (ATH1909) [inline code/modules/economy/account.dm:133]:
```suggestion
		return "A least 1u"
```

Comment (Timberpoes) [reply code/modules/economy/account.dm:133]: Surely it would be "At least 1u"?

Comment (Rohesie) [reply code/modules/economy/account.dm:133]:
```suggestion
		return "At least 1u"
```
no u

**Review by Rohesie — APPROVED** [review]
Use `git stash` to switch between branches without committing.

**Pull request merged**